invocation-count-measurers/src/bench/scala/org/scalameter/examples/ListBenchmark.scala

```diff
@@ -0,0 +1,29 @@
+package org.scalameter.examples
+
+import org.scalameter.api._
+import org.scalameter.execution.invocation.InvocationCountMatcher
+import org.scalameter.execution.invocation.InvocationCountMatcher.{MethodMatcher, ClassMatcher}
+
+
+object ListBenchmark extends PerformanceTest.Microbenchmark {
+  val sizes = Gen.range("size")(300000, 1500000, 300000)
+
+  val lists = for {
+    size <- sizes
+  } yield (0 until size).toList
+
+  performance of "Option" in {
+    measure method "allocations" in {
+      using(lists) config (
+        exec.independentSamples -> 1 // we rather want one JVM instance since this measurer is deterministic
```

Comment:
Btw, I've noticed also that setting:
```exec.independentSamples -> 1
```
with e.g. `OfflineRegressionReport` throws an exception:
```
org.apache.commons.math3.exception.NotStrictlyPositiveException: degrees of freedom (0)
	at org.apache.commons.math3.distribution.TDistribution.<init>(TDistribution.java:92)
	at org.apache.commons.math3.distribution.TDistribution.<init>(TDistribution.java:71)
	at org.apache.commons.math3.distribution.TDistribution.<init>(TDistribution.java:55)
	at org.scalameter.utils.Statistics$.org$scalameter$utils$Statistics$$qt(Statistics.scala:189)
	at org.scalameter.utils.Statistics$.confidenceInterval(Statistics.scala:42)
	at org.scalameter.utils.Statistics$Standard2WayTest.ci1(Statistics.scala:60)
	at org.scalameter.utils.Statistics$OverlapTest.ci1(Statistics.scala:105)
	at org.scalameter.utils.Statistics$OverlapTest.<init>(Statistics.scala:110)
	at org.scalameter.reporting.RegressionReporter$Tester$OverlapIntervals.confidenceInterval(RegressionReporter.scala:370)
	at org.scalameter.reporting.RegressionReporter$Tester$OverlapIntervals.multiple(RegressionReporter.scala:345)
	at org.scalameter.reporting.RegressionReporter$Tester$OverlapIntervals$$anonfun$34.apply(RegressionReporter.scala:360)
	at org.scalameter.reporting.RegressionReporter$Tester$OverlapIntervals$$anonfun$34.apply(RegressionReporter.scala:358)
...
``` 
It it supposed to do so or is it a bug?